dkc/core/tasks.py

```diff
@@ -1,10 +1,15 @@
 from celery import shared_task
 
-from dkc.core.models import File
+from dkc.core.models import File, Folder
 
 
 @shared_task()
 def file_compute_sha512(file_id: int):
     file = File.objects.get(pk=file_id)
     file.compute_sha512()
     file.save()
+
+
+@shared_task()
+def delete_folder(folder_id: int):
+    Folder.objects.get(pk=folder_id).delete()
```

Comment:
Strictly speaking, we should consistently use either `id` or `pk`. Here, it doesn't actually matter, since we know those are aliased to the same column.

Reply:
Could you clarify what you mean? Or perhaps make a more specific recommendation? It's unclear to me when to use each one.